infrastructure/policies.tf

```diff
@@ -109,3 +109,23 @@ resource "aws_iam_policy" "transfer_family_kill_switch" {
     ]
   })
 }
+
+resource "aws_iam_policy" "mtls_access_ssm_policy" {
+  name = "${terraform.workspace}_mtls_ssm_parameters"
+  policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [
+      {
+        Effect = "Allow",
+        Action = [
+          "kms:Encrypt",
```

Comment:
if the lambda's just need to read the encrypted SSM parameter, I think they will only need kms:Decrypt (thought it looks like some PDM policies for reading ssm params give both kms:Decrypt and kms:GenerateDataKey)

Reply:
About the ssm_access_policy (this might be something you want to look at in a separate ticket as i can imagine it being a bigger piece of work) but it could be good to split out the permissions for GetParameter and PutParameter, so only lambda's updating SSM params have the PutParameter permission and ideally for the parameters they use?